schema.sql

```diff
@@ -27,3 +27,28 @@ ALTER TABLE animals ADD COLUMN owners_id INT;
 
 ALTER TABLE animals ADD CONSTRAINT fk_owners FOREIGN KEY (owners_id) REFERENCES owners(id);
 
+-- Create the vets table
+CREATE TABLE vets (
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    name VARCHAR(255),
+    age INT,
+    date_of_graduation DATE
+);
+
+-- Create the specializations table
+CREATE TABLE specializations (
+    vet_id INT,
+    species_id INT,
+    FOREIGN KEY (vet_id) REFERENCES vets(id)
+);
+
+-- Create the visits table
+CREATE TABLE visits (
+    animal VARCHAR(255),
```

Comment:
As requested by the previous reviewer, you do not need the animal varchar column here, but rather the animal_id which should be an `int` just like the vet_id. Kindly consider removing this line and updating it with the animal id (`animal_id int`)